kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/NetworkConfigurationServiceImpl.java

```diff
@@ -92,6 +92,7 @@ public class NetworkConfigurationServiceImpl implements NetworkConfigurationServ
     private static final String CONFIG_AUTOCONNECT = ".config.autoconnect";
     private static final String CONFIG_MTU = ".config.mtu";
     private static final String NET_INTERFACES = "net.interfaces";
+    private static final String MODIFIED_INTERFACE_NAMES = "modified.interface.names";
```

Comment:
You should be able to simply fix the [issue Sonar reported](https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&branch=PR-4242&id=org.eclipse.kura%3Akura&open=AYR8XSFmk5A3ZVwhEoCN) by using this constant.